Examples/Examples.playground/Pages/00ToC.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,20 @@
+/*:
+ ## SwifterSwift Examples
+ 
+ SwifterSwift is a library of **over 500 properties and methods**, designed to extend Swift's functionality and productivity, staying faithful to the original API design guidelines.
+ 
+ You can find examples of some extenstions and try them out in this playground.
+ 
+ Here are list of extensions:
+ 
+ * [SwiftStdlib extensions](01SwiftStdlibExtensions)
```

Comment:
About the naming of the files maybe we can use`01-SwiftStdlibExtensions` instead of `01SwiftStdlibExtensions`